src/css/custom.css

```diff
@@ -524,3 +524,8 @@ div[class^="announcementBar_"] {
 .list-disc > a {
   display: inherit;
 }
+
+/*Correcting alignment of home icon innavbar breadcrumbs*/
+a[class="breadcrumbs__link"]{
+  margin-bottom: -10px;
+}
```

Comment:
Hey @jeromtom , It's slightly lower than the required position. Can you fix that?
Below are the results that I got when I tested the changes.

Desktop site
![Screenshot from 2023-03-05 03-12-04](https://user-images.githubusercontent.com/74777863/222929906-ecfc015d-9d41-4a72-ac97-6888e98a803e.png)

Normal
![Screenshot from 2023-03-05 03-12-55](https://user-images.githubusercontent.com/74777863/222929910-3263d1e6-85e1-4ed6-a7f7-89d392015ed4.png)

Phone
![Screenshot from 2023-03-05 03-13-12](https://user-images.githubusercontent.com/74777863/222929911-1eb93e30-b64d-4580-ab08-5943aa1b710c.png)

Reply:
Will try a margin of -8px